src/tests/lib/components/Modal.spec.ts

```diff
@@ -123,6 +123,19 @@ describe("Modal", () => {
       backdrop && fireEvent.click(backdrop);
     }));
 
+  it("should trigger close modal on Esc", () =>
```

Comment:
Following one suggestion dK made me in another PR, worth adding a test that the modal does not close if one press another key like 'a' or 'b' ?

Reply:
I did it manually, but agree, let's have it in the list.